### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by sdavisde**
Merging dev into master

### CODE REVIEW

**Comment by sdavisde** [comment]
Changes made in this pull request:

General use of plugin:
1. look at teams available for an organization
2. click on team row in table, redirects to that Team's view
3. Look at repo's available for selected Team.
4. Click on repo of choice, redirects to that Repo's view
5. View vulnerability details for that repository 

Organization Page
1. Made two graphs from Nivo library (static data for now)
2. Made a table to output Teams in an organization
3. Made a backend API called useGetTeamsForOrganization to get teams for an org
4. Made a backend API call to get all organizations for a user
5. Made a sub navbar to route between pages (to be removed in the next PR for a more conventional solution)
6. Added clickability / tunneling effect when rows in table are clicked
7. We hard-coded our test organization for now, to be removed in the next PR.

Team Page
1. Made two graphs from Nivo library (static data for now)
2. Made a table to output Repos in a team
3. Made a backend API called useGetRepositoriesForTeam to get repos for a team
4. Added clickability / tunneling effect when rows in table are clicked
5. We hard-coded our test organization for now, to be removed in the next PR.

Repo Page
1. Added four columns to display vulnerabilities by severity level
2. Included details / information for each vulnerability alert given by dependabot
3. Added a switch for viewing OPEN vs All vulnerabilities
4. Made a backend API that retrieves the list of vulnerability alerts with their details for a repository called getVulnerabilitiesForRepo.

We added tests to almost all front end components and included them for almost all backend APIs

File: src/components/Pages/Repo/Repo.test.ts
Comment (granthill7) [inline src/components/Pages/Repo/Repo.test.ts:1]: Is this file being used alongside Repo.test.tsx or are they duplicates?

Comment (granthill7) [inline src/components/Pages/Repo/Repo.test.ts:1]: Is this file being used alongside Repo.test.tsx or are they duplicates?

File: src/components/DataView/index.ts
Comment (granthill7) [inline src/components/DataView/index.ts:1]: Do individual index.ts files need to exist or can the export take place inside of the component? Asking for my own knowledge

**Review by granthill7 — COMMENTED** [review]
This all looks great! I left a couple of comments, if you guys could respond to those we can get this approved.

**Review by EatMoreOranges — APPROVED** [review]

**Pull request merged**